scipy/linalg/_decomp_lu.py

```diff
@@ -45,11 +45,14 @@ def lu_factor(a, overwrite_a=False, check_finite=True):
 
     See Also
     --------
+    lu : gives lu factorization in more user-friendly format
     lu_solve : solve an equation system using the LU factorization of a matrix
 
     Notes
     -----
-    This is a wrapper to the ``*GETRF`` routines from LAPACK.
+    This is a wrapper to the ``*GETRF`` routines from LAPACK. Unlike
+`linalg.lu`, It outputs the L and U factors into a single array,
+and returns pivot indices instead of a permutation matrix.
```

Comment:
```suggestion
    `linalg.lu`, It outputs the L and U factors into a single array,
    and returns pivot indices instead of a permutation matrix.
```